jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc

```diff
@@ -596,12 +624,13 @@ ifndef::env-aws[]
 You will need the following details when you configure CircleCI server.
 
 * *Storage Bucket Name* - The bucket used for server.
+* *Kubernetes Namespace* - The Kubernetes namespace for CircleCI server installation.
 
-* You can choose one of the following:
+* You can choose one of the following options for workload/pod authentication:
```

Comment:
@lloydChris , I see that you decided to remove this part from the AWS bucket section in your [PR](https://github.com/circleci/circleci-docs/pull/6995/files#diff-234110c59a9f2eea648e2f961c2225034458b3ab97213eeb92eeb04ba3bfc988R338).

I've been wondering if this summary is beneficial and I might follow suit and remove it from here as well 🤔

Reply:
Yea. It _was_ a 1:1 realtion to what the customer would see in kots.  But that is not longer tha case.  With the conditionals around rbac vs static keys i dont think it has much value in this section.  The im’ortant part here is that customers are gathering all the info they need, in the next section they get the details of where to put it